web/src/app/components/Modal/EditModal.tsx

```diff
@@ -0,0 +1,32 @@
+import { Modal, Button, Box } from '@mantine/core';
+import { IconXboxX } from '@tabler/icons-react';
+import { ReactNode } from 'react';
+import styles from '../../styles/Modal.module.css';
+
+type ModalProp = {
+  opened: boolean;
+  close: () => void;
+  content: ReactNode;
+};
+
+export default function EditModal({ opened, close, content,title }: ModalProp) {
+  return (
+    <Modal
+      opened={opened}
+      onClose={close}
+      closeButtonProps={{
+        icon: <IconXboxX size={20} stroke={1.5} color="#ffffff" />,
+      }}
+      centered
+      size="100%"
+      classNames={{
+        content: styles.content,
+        body: styles.body,
+        title: styles.title
+      }}
+      title={title}
```

Comment:
The title is not displaying accurately when you switch between modals, i.e. Edit banner to edit Profile. Likely because the title prop is missing from your ModalProp object.